Foundation/NSDictionary.swift

```diff
@@ -264,7 +264,17 @@ public class NSDictionary : NSObject, NSCopying, NSMutableCopying, NSSecureCodin
         return matching
     }
 
-    public override var description: String { NSUnimplemented() }
+    public override var description: String {
+        get {
+            var _description: String = "{\n"
+            for key in self.allKeys {
+                _description += "    \(key) = \(objectForKey(key)!);\n"
```

Comment:
what happens if you have a dictionary of strings to dictionaries? shouldn't this cascade with indentation?